std/bitmanip.d

```diff
@@ -247,12 +248,39 @@ private template createTaggedReference(T, ulong a, string name, Ts...)
 }
 
 /**
-Allows creating bit fields inside $(D_PARAM struct)s and $(D_PARAM
-class)es.
+Allows creating `bitfields` inside `structs`, `classes` and `unions`.
 
-The type of a bit field can be any integral type or enumerated
-type. The most efficient type to store in bitfields is $(D_PARAM
-bool), followed by unsigned types, followed by signed types.
+A `bitfield` consists of one or more entries with a fixed number of
+bits reserved for each. The types of the enties can be `bool`s,
+integral types or enumerated types, arbitrarily mixed.
+
+Integral and Enumerated types are accompanied by two `enums`: `min`
+and `max`, giving the minimum and the maximum number, that can be
+represented by this entry. The variable names are preceded by the
+name of the variable followed by an underscore, so variable `x` comes
+along with `x_min` and `x_max`.
```

Comment:
Aren't `min` and `max` optional? If so, it should be specified. Also, what are the default values?

Reply:
The mixin always adds x_min (etc.) and x_max (etc.) for integeres and enums, but not for bools. I'm not sure, if this could be clearified better.